packages/firebase_vertexai/firebase_vertexai/lib/src/live_session.dart

```diff
@@ -84,7 +84,7 @@ class LiveSession {
   ///
   /// Parameters:
   /// - [mediaChunkStream]: The stream of [InlineDataPart] objects to send to the server.
```

Comment:
[out of scope for this PR] Consider adding a description of the returned Future in the doc. When does it fire? When the sent stream ends?